public/achievementInfo.php

```diff
@@ -224,7 +224,7 @@ function onUpdateEmbedComplete(data) {
             <?php
             echo "<div style='clear:both;'></div>";
 
-            if ($achFlags == 3) {
+            if ($achFlags == 3 && (($permissions >= 3) || ($permissions == 2 && $dataOut['Author'] == $user))) {
```

Comment:
Permission check should only remove the demote link. The "State: Official" text should still be otherwise displayed to match the "State: Unofficial" behavior.

Reply:
Changed to show "State: Unofficial" with no link if Jr Developer and on someone else's achievement.

Reply:
Unless there's a permissions check somewhere else, Unofficial will be shown for everyone. I recommend doing the same for Official. The only change should be hiding the "Demote" link.